### PR TITLE
Update walkthrough-accessing-the-dte-object-from-an-editor-extension.md

### LINKED ISSUE #2163: Missing reference
The SVsServiceProvider class can't be accessed unless you also add a reference to Microsoft.VisualStudio.Shell.Framework. Without it I could not proceed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb4ce0ea-fd74-c3f1-5154-00db7cacab01
* Version Independent ID: 110c664a-cf94-0b5b-4052-3c96c5816452
* Content: [Walkthrough: Accessing the DTE Object from an Editor Extension - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/extensibility/walkthrough-accessing-the-dte-object-from-an-editor-extension?view=vs-2017)
* Content Source: [docs/extensibility/walkthrough-accessing-the-dte-object-from-an-editor-extension.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/extensibility/walkthrough-accessing-the-dte-object-from-an-editor-extension.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gregvanl
* Microsoft Alias: **gregvanl**

### PR DESCRIPTION
**Pull request by lennartb-**
Add missing using directive that needs to be imported.

Fixes #2163

### CODE REVIEW

**Comment by PRMerger12** [comment]
@lennartb- : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

File: docs/extensibility/walkthrough-accessing-the-dte-object-from-an-editor-extension.md [hunk 1]
```diff
@@ -40,8 +40,13 @@ In VSPackages, you can get the DTE object by calling the <xref:Microsoft.VisualS
 
     ```
 
-4. In the `GetDTEProvider` class, import a <xref:Microsoft.VisualStudio.Shell.SVsServiceProvider>.
+4. In the `GetDTEProvider` class and add the following `using` directives:
```
Comment (gewarren) [inline docs/extensibility/walkthrough-accessing-the-dte-object-from-an-editor-extension.md]:
```suggestion
4. In the `GetDTEProvider` class, add the following `using` directive:
```

**Comment by PRMerger8** [comment]
@gewarren : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

**Comment by lennartb-** [comment]
I checked this as well and assumed, since the mentioned `GetClassifier()` method is in a different file (DTETestProvider.cs) than the file the usings were added (DTETest.cs), this would still be necessary. The naming of the classes and implemented methods also seems slightly different in VS 2019 than in the article (which targets VS 2017?), but I didn't want to change that without checking here first.

**Comment by gewarren** [comment]
@lennartb- Ah I see, yes I tried this in VS 2019 and the class is named `EditorClassifier1Provider`. Which version of VS did you try this in?

**Comment by lennartb-** [comment]
Also VS 2019. The class is automatically named <Name>Provider which already differs from docs (GetDTEProvider), and the GetClassifier() method is also implemented as `GetClassifier(ITextBuffer buffer)` instead. I figured this just changed between VS versions, but I currently only have 2019 installed and can't try out the older version. I could check 2015 and 2017 on monday though.

**Comment by gewarren** [comment]
@lennartb- Yes disregard my comment about the class being named `EditorClassifier1Provider`. That was a user error on my part. I'm trying this on both VS 2017 and VS 2019 and will update the docs. Thanks for your help!

**Pull request closed without merging**